etl/collections/base.py

```diff
@@ -0,0 +1,47 @@
+"""WIP: Drafting a model for dealing with MDIM/Explorer configuration.
+
+This should be aligned with the MDIM schema.
+"""
+
+from typing import Any, Dict, List, Optional
+
+
+class Config:
```

Comment:
Eventually, you might want to make a dataclass from it with methods for saving / loading dicts, jsons, etc.

You can check out e.g. [Origin class](https://github.com/owid/etl/blob/a7d34a7bc445600d93091fe5b2f8dcf6359cb2a4/lib/catalog/owid/catalog/meta.py#L110-L112) for inspiration.